src/test/java/com/netflix/playback/features/model/impl/PlayBackFeaturesTest.java

```diff
@@ -0,0 +1,66 @@
+package com.netflix.playback.features.model.impl;
+
+import java.util.concurrent.TimeUnit;
+
+import org.junit.Test;
+
+import com.netflix.playback.features.model.DiagnosticService;
+import com.netflix.playback.features.model.PeriodCompletedCallback;
+import com.netflix.playback.features.model.PlaybackDiagnostics;
+import com.netflix.playback.features.model.PlaybackRequest;
+import com.netflix.playback.features.model.Timer;
+import com.netflix.playback.features.model.impl.AggregatorPeriodCompletedCallback;
+import com.netflix.playback.features.model.impl.DiagnosticServiceImpl;
+import com.netflix.playback.features.model.impl.PlaybackDiagnosticsImpl;
+import com.netflix.playback.features.model.impl.TimerImpl;
+
+import junit.framework.Assert;
+
+public class PlayBackFeaturesTest {
```

Comment:
In terms of test coverage, how might you refactor PeriodAggregator to test the core operations?